lms/djangoapps/teams/csv.py

```diff
@@ -368,8 +370,25 @@ def validate_compatible_enrollment_modes(self, user, team_name, teamset_id):
                 'Team {} cannot have Master’s track users mixed with users in other tracks.'.format(team_name)
             self.add_error_and_check_if_max_exceeded(error_message)
             return False
+        if not self.is_enrollment_protection_for_existing_team_matches_user(user, team_name, teamset_id):
```

Comment:
how is this different than is_FERPA_bubble_breached? I understand how they're different but like... it seems like we wouldn't need both of them, right?

Reply:
Like ... ferpa bubble check is strictly a subset of this check it seems, right? the error message is nicer though, guess it doesn't hurt to keep

Reply:
this is mostly just me thinking out loud

Reply:
in is_FERPA_bubble_breached we don't examine the actual protection mode of the team. We only check enrollment modes of students vs org protected modes. In a way, this is an augmentation to the fix.